README.md

```diff
@@ -1,29 +1,25 @@
-# FirstVoices Unicode Confusables and Adjacency Pair Databases
-
-## What are these databases?
-
-The FirstVoices Unicode 'Confusables' Database is a collection of Unicode characters found primarily within BC-based Indigenous languages, alongside the Unicode characters that can be mistakenly used in place of them.
-
-The FirstVoices Unicode 'Adjacency Pair' Database is an aggregated collection of Unicode-based character pairs used in BC-based Indigenous languages, alongside a snapshot of their type-based frequencies from the FirstVoices language platform. It demonstrates the relative frequency of different Unicode characters as well as the relative frequency of adjacent pairs of characters, across BC-based Indigenous languages. There are three component resources: a database of pairs by Unicode codepoint, a database of pairs by grapheme, and a list of graphemes.
-
-[Read more about these resources and their context on our Knowledge Base.](https://firstvoices.atlassian.net/wiki/spaces/FIR1/pages/1704341)
-
-
-## Why are we publishing these databases?
+# FirstVoices Unicode and Orthography Resources
 
 Indigenous languages are sorely underrepresented online and in digital spaces. UTF-8 is the standard encoding that should be used, as it provides access to all the unicode symbols necessary. Despite that, thorough testing is necessary to make digital environments truly inclusive. We see this work as useful and applicable for digital projects, whether they have an Indigenous focus or not.
```

Comment:
Suggestion from Chris: move this first paragraph to the Knowledge Base and just let these READMEs be pretty technical